end2end-test-examples/gcs/logging.properties

```diff
@@ -7,3 +7,8 @@ io.grpc.ChannelLogger.level = FINEST
 .level = FINE
 io.grpc.netty.NettyClientHandler = ALL
 io.grpc.netty.NettyServerHandler = ALL
+
+com.google.api.client.http.HttpTransport.level=ALL
```

Comment:
Do you still need this or does it serve an additional purpose?

Reply:
It seemed like this logging.properties was configured to do verbose logging for gRPC requests, I've added the corresponding appenders for our HTTP client as well as the shaded version of netty since that's what the gapics use.